core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -963,6 +963,9 @@ abstract class Bundle(implicit compileOptions: CompileOptions) extends Record {
     for (m <- getPublicFields(classOf[Bundle])) {
       getBundleField(m) match {
         case Some(d: Data) =>
+          if (!d.binding.isEmpty)
+            throwException(s"Bundle can only take type elements (got value binding on '${m.getName()}')")
+
```

Comment:
```suggestion
          if (d.binding.nonEmpty) {
            throwException(s"Bundle can only take type elements (got value binding on '${m.getName()}')")
          }

```
Very much a nit but generally prefer `.nonEmpty` to `!_.isEmpty`, also, best to have the curly braces to avoid issues like the "goto fail" bug.